src/repository/todo.repository/types.ts

```diff
@@ -0,0 +1,8 @@
+export type ToDo = {
+  title: string;
+  description: string;
+  createdDate: string;
+  id: string;
+};
+
+export type NewToDoDto = { title: string; description: string };
```

Comment:
DTO is presentation layer, it doesn't belong under `/repository`

Reply:
suggestion where to move it?

Reply:
We could create a folder called dto, under /routes folder, since it's so tightly coupled with routers